modules/50_damages/KotzWenz/input/files

```diff
@@ -0,0 +1,5 @@
+f50_gdp.cs3r
```

Comment:
If I understand it correctly, this is basically the same file, so we might want to drop one of them.
```suggestion
f_gdp.cs3r
```